module/Entra/AdditionalFunctions/Set-EntraAdministrativeUnit.ps1

```diff
@@ -0,0 +1,94 @@
+function Set-EntraAdministrativeUnit {
```

Comment:
Add license information at the top of this file

Reply:
done